languages/python/exercises/concept/dicts/.docs/instructions.md

```diff
@@ -0,0 +1,32 @@
+In this exercise you'll be managing an inventory system.
+
+You will be given a list of items, each time an item is in the given list add `1` to the key in the _given_ inventory. Each item should be organized by their name and the amount of that item. You will also have to delete items from a dictionary.
+
+You will also have to implement a function which returns a list of `tuples` of all the keys in the _given_ inventory.
+
+## 1. Add items from a list to a dictionary
```

Comment:
I'd add an explicit creation step.  Maybe "implement a function that creates an inventory map based on an list"?